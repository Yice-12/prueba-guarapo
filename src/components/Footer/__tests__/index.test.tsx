import { render } from "@testing-library/react";
import { Footer } from "../";

test("render Footer", () => {
  const { asFragment } = render(<Footer />);

  expect(asFragment()).toMatchSnapshot();
});
