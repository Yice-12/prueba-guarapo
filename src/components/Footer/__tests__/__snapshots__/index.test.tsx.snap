// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer 1`] = `
<DocumentFragment>
  <div
    class="footer"
  >
    <div
      class="footer__content"
    >
      <p>
        Make with ❤️ for the MobProgramming team
      </p>
    </div>
  </div>
</DocumentFragment>
`;
